lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -31,6 +35,16 @@
 
                 postRender: function() {
                     // Add subviews
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: new Backbone.Collection(this.context.course_codes),
```

Comment:
I would prefer to move this Collection instantiation to the initialize function. I think it keeps the code a bit cleaner and also enables other subviews to use the collection if they need to.